ovh/ovh.go

```diff
@@ -250,6 +251,19 @@ func (c *Client) getTime() (*time.Time, error) {
 	return &serverTime, nil
 }
 
+// getTarget returns the URL to target given and endpoint and a path.
+// If the path starts with `/v1` or `/v2`, then remove the trailing `/1.0` from the endpoint.
+func getTarget(endpoint, path string) string {
+	// /1.0 + /v1/ or /1.0 + /v2/
+	if (strings.HasSuffix(endpoint, "/1.0") && (strings.HasPrefix(path, "/v1/") || strings.HasPrefix(path, "/v2/"))) ||
```

Comment:
I think we could simplify like this: 
```
if (strings.HasSuffix(endpoint, "/1.0") || strings.HasSuffix(endpoint, "/1.0/")) && (strings.HasPrefix(path, "/v1/") || strings.HasPrefix(path, "/v2/"))
```
And use path.Join to concatenate the endpoint and the path

Reply:
Another way of handling this would be: let's forbid final slash on endpoint definition.
Regular user should use endpoint defined by us, defined in `ovh.go` 

That would prevent users to use `client.Get("dedicated/server")` which is weird to me.